### ISSUE TITLE
Allow clearing initially set values

### ISSUE THREAD

**Issue by inikolova** [telerik/kendo-ui-core#1883]
Provide option for clearing ilitially set values. For example **valueAxis.majorUnit** - once set it cannot be cleared (http://dojo.telerik.com/@Iliana/UkOTO)

**Comment by danielkaradachki** [comment]
closed with telerik/kendo@793094fc26b83fdf99754357aaba52622e10001a

**Comment by tsvetomir** [comment]
For example:
```
chart.setOptions({
    valueAxis: { majorUnit: null }
});
``
